src/app/App.js

```diff
@@ -44,6 +47,7 @@ define([
 
     Print,
     Extent,
+    ArcGISDynamicMapServiceLayer,
```

Comment:
this should probably be a [feature layer](https://developers.arcgis.com/javascript/3/jsapi/featurelayer-amd.html)